From c79a302c3ddf91ef9b389be3bab47abf4261e2e3 Mon Sep 17 00:00:00 2001
From: Samuel Cabrero <scabrero@suse.de>
Date: Mon, 14 Jan 2019 13:06:55 +0100
Subject: [PATCH 2/9] krb5-1.9-manpaths

Import krb5-1.9-manpaths.dif

Change the absolute paths included in the man pages so that the correct
values can be dropped in by config.status.  After applying this patch,
these files should be renamed to their ".in" counterparts, and then the
configure scripts should be rebuilt.  Originally RT#6525
---
 src/man/kpropd.man | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/man/kpropd.man b/src/man/kpropd.man
index b1caad000..9bfa7be94 100644
--- a/src/man/kpropd.man
+++ b/src/man/kpropd.man
@@ -67,7 +67,7 @@ the \fB/etc/inetd.conf\fP file which looks like this:
 .INDENT 3.5
 .sp
 .EX
-kprop  stream  tcp  nowait  root  /usr/local/sbin/kpropd  kpropd
+kprop  stream  tcp  nowait  root  @SBINDIR@/kpropd  kpropd
 .EE
 .UNINDENT
 .UNINDENT
-- 
2.51.0

